mappings/net/minecraft/world/gen/foliage/BrushFoliagePlacer.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_5205 net/minecraft/world/gen/foliage/BrushFoliagePlacer
```

Comment:
Not `Bush`?

Reply:
I'll double-check, but I'm pretty sure that's what it's registered as.